integration/docker/package_manager_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("package manager update test", func() {
+	var (
+		id         string
+		args       []string
+		proxyVar   string
+		proxyValue string
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		args = []string{}
+		proxyVar = "http_proxy"
+		proxyValue = os.Getenv(proxyVar)
+		if proxyValue != "" {
+			args = append(args, "-e", proxyVar+"="+proxyValue)
+		}
+	})
+
+	AfterEach(func() {
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	Context("check apt-get update", func() {
+		It("should not fail", func() {
+			args = append(args, "--rm", "--name", id, DebianImage, "apt-get", "-y", "update")
+			_, _, exitCode := dockerRun(args...)
+			Expect(exitCode).To(BeZero())
+		})
+	})
+
+	Context("check dnf update", func() {
+		It("should not fail", func() {
+			Skip("Issue: https://github.com/clearcontainers/runtime/issues/868")
```

Comment:
This raises an interesting question: should this skip only apply to `cc-runtime` (when run as `kata-runtime`)? Although `runv` might have the same limitation, I think we will potentially need a way to skip based on runtime. I've raised #59 on what might be an interesting little project.

Reply:
seems like at some point yes, we would need to apply the skip based on the runtime. Thanks for raising the issue.